com.sap.cloud.lm.sl.cf.client/src/main/java/com/sap/cloud/lm/sl/cf/client/util/StreamUtil.java

```diff
@@ -1,75 +1,55 @@
 package com.sap.cloud.lm.sl.cf.client.util;
 
 import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.FileOutputStream;
 import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.text.MessageFormat;
 import java.util.Arrays;
 import java.util.zip.ZipEntry;
-import java.util.zip.ZipInputStream;
 
 import org.apache.commons.io.FileUtils;
 import org.apache.commons.io.FilenameUtils;
-import org.apache.commons.io.IOUtils;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
-public class StreamUtil {
+import com.sap.cloud.lm.sl.cf.client.message.Messages;
+import com.sap.cloud.lm.sl.common.ContentException;
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(StreamUtil.class);
+public class StreamUtil {
 
     static final String PATH_SHOULD_NOT_BE_ABSOLUTE = "Archive entry name \"{0}\" should not be absolute";
     static final String PATH_SHOULD_BE_NORMALIZED = "Archive entry name \"{0}\" should be normalized";
-    static final String ARCHIVE_ENTRY_SEPARATOR = "/";
+    public static final String ARCHIVE_ENTRY_SEPARATOR = "/";
 
-    public static File saveStream(String fileName, InputStream inputStream) throws IOException {
+    public static boolean isArchiveEntryDirectory(String fileName) {
         if (fileName.endsWith(ARCHIVE_ENTRY_SEPARATOR)) {
-            return saveZipStreamToDirectory(fileName, inputStream);
-        } else {
-            return saveStreamToFile(fileName, inputStream);
+            return true;

```

Comment:
Maybe it would be cleaner if you just return "fileName.endsWith(ARCHIVE_ENTRY_SEPARATOR"